Sources/AWSSDKSwiftCore/Message/Body.swift

```diff
@@ -31,47 +29,34 @@ extension Body {
     /// initialize Body with Any. If it is Data, create .buffer() otherwise create a String describing the value
     init(anyValue: Any) {
```

Comment:
Why do we have an initializer with `Any`? What are the usecases? That looks a little spooky to me.

Reply:
Yeah, I kinda stumbled on that one as well. The situation is we have a payload which is an arbituary object, could be a string, could be data. It is accessed through `Mirror`. The next stage of this work (the Payload object) may mean we can remove this.

https://github.com/swift-aws/aws-sdk-swift-core/blob/0dc610d31f25b0252bb5cbdfb8b76eb3cdfdd7b1/Sources/AWSSDKSwiftCore/AWSClient.swift#L383

Reply:
Okay, do we want to open an issue for this?

Reply:
This is the work I was talking about https://github.com/swift-aws/aws-sdk-swift-core/issues/185

Reply:
Basically all non AWSShape payloads would be managed by this Payload enum. It is currently an enum but we could convert it to a wrapper for ByteBuffer but I had an idea we could also use it to inject streamed payloads as well.

Reply:
Added #211 as #185 doesn't detail cleaning up of this init() function

Reply:
You happy for this to go in now?